clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/s3/S3ArtifactCredentials.java

```diff
@@ -56,6 +63,9 @@ protected AmazonS3 getS3Client() {
       builder.setPathStyleAccessEnabled(true);
     } else if (!StringUtils.isEmpty(region)) {
       builder.setRegion(region);
+    } else if (!StringUtils.isEmpty(awsAccessKeyId) && !StringUtils.isEmpty(awsSecretAccessKey)) {
```

Comment:
I think this should go outside the `if/else` block. The above is used to differentiate between _real_ S3 and clones like Minio. Static creds still apply in both cases.